tests/scripts/github-action-helper.sh

```diff
@@ -294,13 +303,29 @@ function deploy_cluster() {
     echo "invalid argument: $*" >&2
     exit 1
   fi
+
   # enable monitoring
   yq w -i -d0 cluster-test.yaml spec.monitoring.enabled true
   kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.71.1/bundle.yaml
   kubectl create -f monitoring/rbac.yaml
 
-  # create the cluster resources
   kubectl create -f cluster-test.yaml
+
+  deploy_toolbox
+}
+
+# These resources were extracted from the original deploy_cluster(), which was
+# deploying a smorgasbord of resources used by multiple canary tests.
+#
+# Use of this function is discouraged!  Existing users should migrate to
+# deploy_cluster() and deploy only the necessary resources for the test
+# scenario in their own job steps.
+function deploy_all_resources() {
```

Comment:
What if we instead only deploy the additional resources? And whoever wants to deploy all resources must call both `deploy_ cluster` and this function?
```suggestion
function deploy_all_additional_resources_on_cluster() {
```